src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -98,47 +70,43 @@ export const mhvToolName = appId => {
     case widgetTypes.VIEW_APPOINTMENTS:
       return 'VA Appointments';
 
-    case widgetTypes.DIRECT_DEPOSIT:
-      return 'Direct Deposit';
-
     default: // Not a recognized health tool.
   }
 
   return null;
 };
 
-export const toolUrl = appId => {
+export const toolUrl = (appId, useSSOe = false) => {
   switch (appId) {
     case widgetTypes.HEALTH_RECORDS:
       return {
-        url: mhvUrl('download-my-data'),
+        url: mhvUrl(useSSOe, 'download-my-data'),
         redirect: false,
       };
 
     case widgetTypes.RX:
       return {
-        url: mhvUrl('web/myhealthevet/refill-prescriptions'),
-        redirect: true,
+        url: mhvUrl(useSSOe, 'web/myhealthevet/refill-prescriptions'),
+        redirect: false,
```

Comment:
@patrickvinograd what's the reason for changing the redirect flag from 'true' to 'false'?

Reply:
The main effect of the `redirect` flag is that if enabled and all conditions are met for the CTA widget via the "isAccessible" check, it auto-opens the target tool in a new tab as soon as the page is loaded. But for the MHV tools, the "isAccessible" check was not really sufficient - it was not performing all the business logic around deactivated accounts, etc. So there were users who by rights did not have access, and the CTA widget was saying so, but _also_ it was opening MHV in a new tab. It seemed better to require the user to click the button and have consistent allowed/disallowed logic/presentation.

Reply:
sounds good, ill update the test case.